src/collective/faq/configure.zcml

```diff
@@ -15,14 +15,32 @@
   <include file="upgrades.zcml" />
 
   <genericsetup:registerProfile
-      name="default"
+      name="install-base"
       title="FAQ (collective.faq)"
-      directory="profiles/default"
+      directory="profiles/base"
       description="Installs the FAQ add-on."
       provides="Products.GenericSetup.interfaces.EXTENSION"
       post_handler=".setuphandlers.post_install"
       />
 
+  <genericsetup:registerProfile
```

Comment:
@laulaz wondering if we need an upgrade step here when we switch the gs profile architecture.

Reply:
@tisto I asked myself exactly the same question and though that if `collective.faq` was already installed, as I didn't change the profile version, it wouldn't hurt (and "default" profile still exists).

But I haven't tested all the upgrade paths.

I could write an upgrade step for Plone 5.0 & 5.1 (versions where we come from [css/js]registry to Resource Registry) (?) ... or maybe we go on and drop support for Plone 4 on this branch :innocent: :wink:

Reply:
@laulaz I guess we should rather drop Plone 4 support on this branch. It is not worth the effort and we will have to drop it sooner or later. I am happy to cut a new Plone 5.2 / Python 3 only release.

Reply:
@tisto OK, I will propose the according changes 👍